runtime/terms.go

```diff
@@ -41,6 +41,8 @@ type terms struct {
 
 	// mu protects the state of the terms chain
 	mu sync.Mutex
+	// counts executions
+	counter int
```

Comment:
I think it should be a field/attribute of [Failpoint](https://github.com/etcd-io/gofail/blob/f7a29be7af9858bc75453c26090b14e7d464a691/runtime/failpoint.go#LL21C6-L21C15), and we need to clearly define what does counter exactly mean. There are two options:
- The count of the failpoint execution, no matter the terms are evaluated or not. 
- The count of the failpoint execution, only include the case that the terms are really evaluated. For example, `40.0%return(true)` means 40% possibility to return `true`. If it returns false (falls into the other 60% possibility), then we don't increment the `counter`. 

Your current implementation should be the second option.

Reply:
I have updated the definition of counter. Regarding making it a field of Failpoint, I have the following concerns

1. There are number of _similar_ fields that are defined in terms rather than in failpoint (e.g., [fpath, desc](https://github.com/etcd-io/gofail/blob/f7a29be7af9858bc75453c26090b14e7d464a691/runtime/terms.go#L37-L40)), should we standardize all, rather than moving just one field into Failpoint ?
2. This will involve changes to parsing code, get Failpoint lock and reference in terms.go. 

Should these be addressed as a refactor in a separate PR ?